src/modules/stats/overwatch-stats.ts

```diff
@@ -0,0 +1,189 @@
+import { stripIndents } from 'common-tags';
+import { Message, MessageEmbed } from 'discord.js';
+import { Command, CommandoMessage, CommandoClient } from 'discord.js-commando';
+import { startTyping, stopTyping, deleteCommandMessages } from '../../lib/helpers';
+import { getProfile, Profile, PLATFORM, REGION } from 'overwatch-api';
+
+class OverwatchStatsArguments { 
+	battletag: string; 
+	platform: PLATFORM;
+	region: REGION;
+}
+/**
+ * Post information about an alcoholic brew.
+ *
+ * @export
+ * @class OverwatchStatsCommand
+ * @extends {Command}
+ */
+export default class OverwatchStatsCommand extends Command {
+	/**
+	 * Creates an instance of BrewCommand.
+	 *
+	 * @param {CommandoClient} client
+	 * @memberof BrewCommand
+	 */
+	constructor(client: CommandoClient) {
+		super(client, {
+			args: [
```

Comment:
please change this to the standardized format of args i laid out in https://github.com/Spudnik-Group/Spudnik/pull/212/files#diff-7f79855d668a663c5d5070880